nautobot/extras/tests/test_api.py

```diff
@@ -2179,6 +2181,65 @@ def setUpTestData(cls):
         Tag.objects.create(name="Tag 2", slug="tag-2")
         Tag.objects.create(name="Tag 3", slug="tag-3")
 
+    def test_all_relevant_content_types_assigned_to_tags_with_empty_content_types(self):
+        self.add_permissions("extras.add_tag")
+
+        self.client.post(self._get_list_url(), self.create_data[0], format="json", **self.header)
+
+        tag = Tag.objects.get(slug=self.create_data[0]["slug"])
+        self.assertEqual(
+            tag.content_types.count(),
+            TaggableClassesQuery().as_queryset.count(),
+        )
+
+
+class TagTestVersion13(
+    APIViewTestCases.CreateObjectViewTestCase,
+    APIViewTestCases.UpdateObjectViewTestCase,
+):
+    model = Tag
+    brief_fields = ["color", "display", "id", "name", "slug", "url"]
+    api_version = "1.3"
+    create_data = [
+        {"name": "Tag 4", "slug": "tag-4", "content_types": [Site._meta.label_lower]},
+        {"name": "Tag 5", "slug": "tag-5", "content_types": [Site._meta.label_lower]},
+        {"name": "Tag 6", "slug": "tag-6", "content_types": [Site._meta.label_lower]},
+    ]
+    bulk_update_data = {"content_types": [Site._meta.label_lower]}
+
+    @classmethod
+    def setUpTestData(cls):
+        Tag.objects.create(name="Tag 1", slug="tag-1")
+        Tag.objects.create(name="Tag 2", slug="tag-2")
+        Tag.objects.create(name="Tag 3", slug="tag-3")
```

Comment:
Don't these need to have a content_types value to be valid now?

Reply:
TODO: Add content types.